<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information defense</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLab1">
       <attribute name="title">
        <string>LW 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="9" column="0">
         <widget class="QPushButton" name="btnEncrypt">
          <property name="text">
           <string>Encode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Source data</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Message</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2" colspan="2">
            <widget class="QPlainTextEdit" name="editMessage">
             <property name="plainText">
              <string>ГРУЗИТЕАПЕЛЬСИНЫ</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="lblRequiredLength">
             <property name="text">
              <string>Required length: Unknown</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelInputLength">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Text length: 0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkRemoveSpaces">
             <property name="text">
              <string>Delete spaces</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Output data</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Result</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="editResult"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="btnDecrypt">
          <property name="text">
           <string> Decode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Encryption keys</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0" colspan="3">
            <widget class="QLineEdit" name="editFirstKey">
             <property name="text">
              <string>4132</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QLineEdit" name="editSecondKey">
             <property name="text">
              <string>3142</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>First key</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lblKey1Length">
             <property name="text">
              <string>Length: 0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Second key</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="lblKey2Length">
             <property name="text">
              <string>Length: 0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLab2">
       <attribute name="title">
        <string>LW 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
